docker-compose.yml

```diff
@@ -214,12 +214,16 @@ services:
       - "8126:8126"
 
   # https://hub.docker.com/_/elasticsearch/
-  # Note: This image is deprecated, but the new one requires fiddling.
-  elasticsearch:
+  legacy-elasticsearch:
     image: mozilla/socorro_elasticsearch:1.4.5
     ports:
       - "9200:9200"
-      - "9300:9300"
```

Comment:
IIUC this is used primarily for internode communication within an ES cluster. I was reading about some reasons for why one may or may not want to expose it.

The `git blame` [commit](https://github.com/mozilla-services/socorro/commit/fb9fcae74f13c02fb826a36ce9c34df99878dc41) doesn't mention anything specific about it when it was initially added. Any idea why it may have been, and why it's safe to remove now?

Reply:
i learned this while setting up our elasticsearch cluster in GCP! elasticsearch uses 9300 only for inter-node communication, such as data transfer and master selection (but *not* node discovery), and we only have one node in each dev "cluster", so we're guaranteed not to use it.

Reply:
Thanks! From the ES class I'm taking, IIRC 1 node is the default now for the ES cluster when it starts up, so I guess there wouldn't be an explicit config for this for dev, right? How is this config set for stage and prod though, which I imagine have multiple nodes?

Reply:
we use docker-compose.yml for dev, and we're currently using a helm chart for es 1.4 for stage and prod, which creates an elasticsearch.yml file with configs. we will probably be using terraform to deploy es 8 via elastic cloud in stage and prod.

becaase docker-compose.yml is only used for dev, and dev is configured to use single nodes, we don't need 9300 here.